twake/backend/node/src/services/general/constants.ts

```diff
@@ -0,0 +1,14 @@
+import { Languages, ServerConfiguration } from "./types";
+
+export const languages: Languages = {
+  default: "en",
+  availables: ["en", "fr", "es", "vn", "ru"],
+};
+
+export const version: ServerConfiguration["version"] = {
+  current: /* @VERSION_DETAIL */ "2021.Q2.505",
+  minimal: {
+    web: /* @MIN_VERSION_WEB */ "2021.Q1.481",
+    mobile: /* @MIN_VERSION_MOBILE */ "2021.Q1.385",
+  },
+};
```

Comment:
Why don't we put this in the configuration file?

Reply:
Because this is not configuration but directly related to the code itself (the version of the backend). If I put it in configuration we'll need to change it manually on each update.

Reply:
We can put it in default.json or version.json (it will not be changed by administrator)